docs/user-guides/desktop/index.md

```diff
@@ -129,28 +129,64 @@ While active, Coder Connect will list the workspaces you own and will configure
-   ping6 -c 5 your-workspace.coder
+   ssh your-workspace.coder
    ```
 
-On Windows, you can use `ping` in a Command Prompt or PowerShell terminal to verify the connection to your workspace:
+Any services listening on ports in your workspace will be available on the same hostname. For example, you can access a web server on port `8080` by visiting `http://your-workspace.coder:8080` in your browser.
+
+> [!NOTE]
+> Currently, the Coder IDE extensions for VSCode and JetBrains create their own tunnel and do not utilize the Coder Connect tunnel to connect to workspaces.
+
+### Ping your workspace
+
+<div class="tabs">
+
+### macOS
+
+Use `ping6` in your terminal to verify the connection to your workspace:
 
    ```shell
-   ping -n 5 your-workspace.coder
+   ping6 -c 5 your-workspace.coder
    ```
 
-Any services listening on ports in your workspace will be available on the same hostname. For example, you can access a web server on port `8080` by visiting `http://your-workspace.coder:8080` in your browser.
+### Windows
 
-You can also connect to the SSH server in your workspace using any SSH client, such as OpenSSH or PuTTY:
+Use `ping` in a Command Prompt or PowerShell terminal to verify the connection to your workspace:
 
    ```shell
-   ssh your-workspace.coder
+   ping -n 5 your-workspace.coder
    ```
 
-> [!NOTE]
-> Currently, the Coder IDE extensions for VSCode and JetBrains create their own tunnel and do not utilize the Coder Connect tunnel to connect to workspaces.
+</div>
+
+## Sync a local directory with your workspace
+
+Coder Desktop file sync provides bidirectional synchronization between a local directory and your workspace.
+You can work offline, add screenshots to documentation, or use local development tools while keeping your files in sync with your workspace.
+
+1. Create a new local directory.
+
+   If you select an existing clone of your repository, Desktop will recognize it as conflicting files.
+
+1. In the Coder Desktop app, select **File sync**.
+
+   ![Coder Desktop File Sync screen](../../images/user-guides/desktop/coder-desktop-file-sync.png)
+
+1. Select the **+** in the corner to select the local path, workspace, and remote path, then select **Add**:
+
+   ![Coder Desktop File Sync add paths](../../images/user-guides/desktop/coder-desktop-file-sync-add.png)
+
+1. File sync clones your workspace directory to your local directory, then watches for changes:
+
+   ![Coder Desktop File Sync watching](../../images/user-guides/desktop/coder-desktop-file-sync-watching.png)
+
+File sync excludes version control system directories like `.git/` from synchronization, so keep your Git-cloned repository wherever you run Git commands.
+This means that if you use an IDE with a built-in terminal to edit files on your remote workspace, that should be the Git clone and your local directory should be for file syncs.
+
+If you encounter a synchronization conflict, delete the conflicting file that contains changes you don't want to keep.
```

Comment:
Thoughts on adding a Windows section here, too?

Reply:
Can we also add a note or two on what Ethan described here in the [comment](https://github.com/coder/coder/pull/17463#issuecomment-2814607297)?

Reply:
the mutagen stuff?

I guess we could - I skipped it for user-facing documentation. it's good for us to know how it works or why it could change one day, but there doesn't seem to be anything a Desktop user could do about it. Is there? Mutagen docs @ethanndickson linked to say that the git stuff should be ignored, so even if there is a workaround, we're probably safer not suggesting it